scripts/setupClient.sh

```diff
@@ -56,6 +56,10 @@ echo "command=\"./crypt-scripts/retrieve_"$HOSTNAME"_key\" `cat /etc/initramfs-t
 read
 # Retrieve keyfile
 ssh $keyHost -i /etc/initramfs-tools/root/.ssh/unlock_rsa -o UserKnownHostsFile=/etc/initramfs-tools/root/.ssh/known_hosts "cat .keyfiles/$HOSTNAME" > ./tmp-mount/keyfile
+# Verify keyfile
+echo "Please verify this hash for the keyfile to be correct. Abort, if they differ! (ecf255377f0f44003b40897ca41696c82fbe25dd would mean that MAC-Address verification has failed.)"
+sha1sum ./tmp-mount/keyfile | cut -c -40
```

Comment:
I'd suggest to go with either:
- awk '{ print $1 }'
- cut -d' ' -f1
- anything else that does not depend on hash length.

This way we could switch to sha512sum or whatever else without any problems.

Reply:
OK. I'd prefer `cut`, since this is available on stock Android, too. I'm going to change the instances in the Android branch as well.